e2e-tests/tests/monitoringconfig_pxc/01-assert.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: kuttl.dev/v1
+kind: TestAssert
+timeout: 300
+---
+apiVersion: everest.percona.com/v1alpha1
+kind: DatabaseCluster
+metadata:
+  name: pxc-mc
+spec:
+  monitoringConfigName: test-mc
+status:
+  status: ready
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: everest-secrets-pxc-mc
+data:
+  pmmserverkey: YWJjZGVm
+---
+apiVersion: pxc.percona.com/v1
+kind: PerconaXtraDBCluster
+metadata:
+  name: pxc-mc
+spec:
+  secrets:
+    users: everest-secrets-pxc-mc
```

Comment:
The `MonitoringConfig` CR is missing in the e2e tests asserts

Reply:
Yes I think you're right Michal, I actually thought about this before because we have this across all other tests.
Currently, there isn't much point in testing what you applied.
However, if we end up adding logic to change the spec and add for example the default values we might have a use case for also testing the resource you just applied. This is just far fetched right now.